examples/unions/unions.bal

```diff
@@ -19,14 +19,73 @@ public function main() {
 
     io:println(nameToString(name1));
     io:println(nameToString(name2));
+
+    map<string?> grades1 = {
```

Comment:
```suggestion
    map<string|()> grades1 = {
```
Suggesting this only because we are talking about unions here (We do have a `Nil` BBE separately)

Reply:
Ah, yes. Updated everywhere in 209f026.